generate_co_occurrence_vectors_flow.py

```diff
@@ -103,33 +101,47 @@ def build_index(self):
         from annoy import AnnoyIndex
         from tempfile import NamedTemporaryFile
 
-        with NamedTemporaryFile(delete=False) as tmp:
+        with NamedTemporaryFile() as tmp:
             ann = AnnoyIndex(self.embedding_size, "angular")
             ann.on_disk_build(tmp.name)
             with profile("Add vectors"):
                 for idx, card_vector in enumerate(self.embeddings):
-                    ann.add_item(self.id_to_card[idx], card_vector)
```

Comment:
The wrong move was here, Annoy does not perform well with large IDs: https://github.com/spotify/annoy/issues/438#issuecomment-638967477